docs/config_options.rst

```diff
@@ -3,43 +3,143 @@ Configuration Options
+.. list-table:: Configuration Parameters
+   :widths: 30 50 30 20 20
+   :header-rows: 1
+
+   * - Key(s)
+     - Description
+     - Type
+     - Required
+     - Default
+
+       .. _config_logger:
+   * - ``logger`` :ref:`🔗 <config_logger>`
+     - Most log messages are sent here.
+     - ``logging.Logger``
+     -
+     -
+
+       .. _config_logprint:
+   * - ``logprint`` :ref:`🔗 <config_logprint>`
+     - Messages about available tilesources are sent here.
+     - ``logging.Logger``
+     -
+     -
+
+       .. _config_cache_backend:
+   * - ``cache_backend`` :ref:`🔗 <config_cache_backend>`
+     - String specifying how tiles are cached.  If memcached is not available for any reason, the python cache is used instead.
+     - ``str: "python" | "memcached"``  # What about redis?
+     - ``False``
+     - ``"python"``
+
+       .. _config_cache_python_memory_portion:
+   * - ``cache_python_memory_portion`` :ref:`🔗 <config_cache_python_memory_portion>`
+     - If tiles are cached with python, the cache is sized so that it is expected to use less than 1 / (``cache_python_memory_portion``) of the available memory.
+     - ``int``
+     -
+     -
+
+       .. _config_cache_memcached_url:
+   * - ``cache_memcached_url`` :ref:`🔗 <config_cache_memcached_url>`
+     - If tiles are cached in memcached, the url or list of urls where the memcached server is located.
+     - ``str | List[str]``
+     - ``False``
+     - ``"127.0.0.1"``
+
+       .. _config_cache_memcached_username:
+   * - ``cache_memcached_username`` :ref:`🔗 <config_cache_memcached_username>`
+     - A username for the memcached server.
+     -
+     -
+     - ``None``
+
+       .. _config_cache_memcached_password:
+   * - ``cache_memcached_password`` :ref:`🔗 <config_cache_memcached_password>`
+     - A password for the memcached server.
+     -
+     -
+     - ``None``
+
+       .. _config_cache_redis_url:
+   * - ``cache_redis_url`` :ref:`🔗 <config_cache_redis_url>`
+     - If tiles are cached in redis, the url or list of urls where the redis server is located.
+     - ``str | List[str]``
+     -
+     - ``"127.0.0.1:6379"``
+
+       .. _config_cache_redis_username:
+   * - ``cache_redis_username`` :ref:`🔗 <config_cache_redis_username>`
+     - A username for the redis server.
+     -
+     -
+     - ``None``
+
+       .. _config_cache_redis_password:
+   * - ``cache_redis_password`` :ref:`🔗 <config_cache_redis_password>`
+     - A password for the redis server.
+     -
+     -
+     - ``None``
+
+       .. _config_cache_tilesource_memory_portion:
+   * - ``cache_tilesource_memory_portion`` :ref:`🔗 <config_cache_tilesource_memory_portion>`
+     - Tilesources are cached on open so that subsequent accesses can be faster.  These use file handles and memory.  This limits the maximum based on a memory estimation and using no more than 1 / (``cache_tilesource_memory_portion``) of the available memory.
+     - ``int``
+     -
+     -
```

Comment:
Memory usage by tile source is necessarily a rough estimate, since it can vary due to a wide variety of image-specific and deployment-specific details; this is intended to be conservative.  Default is 32.